### PR TITLE
Fixed text describing Mendix Dedicated

### PR DESCRIPTION
**Pull request by MaartenMcFly**
Mendix Dedicated does not run in a customer's VPC.

### CODE REVIEW

File: content/en/docs/developerportal/deploy/_index.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ For more information, see [Mendix Cloud](/developerportal/deploy/mendix-cloud-de
 
 ### 3.1 Mendix Cloud Dedicated {#mendix-cloud-dedicated}
 
-Organizations can also have their own Mendix Cloud, named [Mendix Cloud Dedicated](https://www.mendix.com/evaluation-guide/app-lifecycle/mendix-cloud-overview/#mendix-cloud-vpc). This works in exactly the same way as the Mendix Cloud, but runs on your own virtual private cloud and will only host your company's apps.
+Organizations can also have their own Mendix Cloud, named [Mendix Cloud Dedicated](https://www.mendix.com/evaluation-guide/app-lifecycle/mendix-cloud-overview/#mendix-cloud-vpc). This works in exactly the same way as the Mendix Cloud except that the cluster on which the apps run is not shared with any other customers. Only your own organization's apps run on this cluster. Furthermore, you have the ability to link this cluster to your backend.
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/deploy/_index.md]: @MaartenMcFly 
Thanks for the PR, Maarten. I just have one question.

What exactly do you mean by having the ability to link the cluster to the customer's backend? What is a use case for this? I'm not sure I would understand exactly what was meant if I was a customer, so perhaps we can make it more specific with an example?

**Comment by MaartenMcFly** [comment]
Hi Mark,

This pertains to direct links to a customer system like a database through VPN. We don't allow this on our public cloud, but for dedicated we allow this. So direct access to a customer's other system over their own network.

Cheers,

Maarten

________________________________
Van: Mark van Ments ***@***.***>
Verzonden: vrijdag 16 juni 2023 10:48
Aan: mendix/docs ***@***.***>
CC: Maarten ***@***.***>; Mention ***@***.***>
Onderwerp: Re: [mendix/docs] Fixed text describing Mendix Dedicated (PR #6188)


@MarkvanMents commented on this pull request.

________________________________

In content/en/docs/developerportal/deploy/_index.md<https://github.com/mendix/docs/pull/6188#discussion_r1231971077>:

> @@ -46,7 +46,7 @@ For more information, see [Mendix Cloud](/developerportal/deploy/mendix-cloud-de

 ### 3.1 Mendix Cloud Dedicated {#mendix-cloud-dedicated}

-Organizations can also have their own Mendix Cloud, named [Mendix Cloud Dedicated](https://www.mendix.com/evaluation-guide/app-lifecycle/mendix-cloud-overview/#mendix-cloud-vpc). This works in exactly the same way as the Mendix Cloud, but runs on your own virtual private cloud and will only host your company's apps.
+Organizations can also have their own Mendix Cloud, named [Mendix Cloud Dedicated](https://www.mendix.com/evaluation-guide/app-lifecycle/mendix-cloud-overview/#mendix-cloud-vpc). This works in exactly the same way as the Mendix Cloud except that the cluster on which the apps run is not shared with any other customers. Only your own organization's apps run on this cluster. Furthermore, you have the ability to link this cluster to your backend.


@MaartenMcFly<https://github.com/MaartenMcFly>
Thanks for the PR, Maarten. I just have one question.

What exactly do you mean by having the ability to link the cluster to the customer's backend? What is a use case for this? I'm not sure I would understand exactly what was meant if I was a customer, so perhaps we can make it more specific with an example?

**Comment by MarkvanMents** [comment]
@MaartenMcFly 
> direct access to a customer's other system over their own network

That is a good phrase. I'll use it instead of "link the cluster to the customer's backend" as I think it is clearer.
Does that seem good, and is it OK to publish immediately once I've made that change?

**Comment by MarkvanMents** [comment]
@MaartenMcFly 
Merging this PR - let me know if you want any other changes.
Thanks for creating it and providing the update to the text.
Mark

**Review by MarkvanMents — APPROVED** [review]

**Comment by MaartenMcFly** [comment]
Good to go with that, no further changes!

**Pull request merged**